diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter2/comments/CommentsRule.kt

```diff
@@ -80,7 +90,8 @@ class CommentsRule(private val configRules: List<RulesConfig>) : Rule("comments"
                     .isEmpty()
             }
             .forEach { (offset, parsedNode) ->
-                COMMENTED_OUT_CODE.warn(configRules, emitWarn, isFixMode, parsedNode.text.substringBefore("\n").trim(), offset, parsedNode)
+                COMMENTED_OUT_CODE.warn(configRules, emitWarn, isFixMode, parsedNode.text.substringBefore("\n").trim(), offset,
+                    errorNodesWithText.find { it.second == parsedNode.text }?.first ?: parsedNode)
```

Comment:
Is this a case for nodes that have not been parsed because do not contain code markers? If I understand correctly, in this case `parsedNode` will be `null`, do you really need to find by text?

Reply:
No, this case for `Suppress`. `parsedNode` is a new node, that we have created, it hasn't parent, so method `hasSuppress` doesn't work.

Reply:
Oh, OK, I see